java/dagger/internal/codegen/CodeBlocks.java

```diff
@@ -25,64 +25,44 @@
 
 import com.google.auto.common.MoreElements;
 import com.google.auto.common.MoreTypes;
-import com.google.errorprone.annotations.CanIgnoreReturnValue;
 import com.squareup.javapoet.ClassName;
 import com.squareup.javapoet.CodeBlock;
-import com.squareup.javapoet.CodeBlock.Builder;
 import com.squareup.javapoet.MethodSpec;
 import com.squareup.javapoet.TypeName;
 import java.util.stream.Collector;
 import javax.lang.model.element.ExecutableElement;
-import javax.lang.model.element.VariableElement;
 import javax.lang.model.type.DeclaredType;
+import javax.lang.model.type.TypeMirror;
 
 final class CodeBlocks {
-  /**
-   * A {@link Collector} implementation that joins {@link CodeBlock} instances together into one
-   * separated by {@code delimiter}. For example, joining {@code String s}, {@code Object o} and
-   * {@code int i} using {@code ", "} would produce {@code String s, Object o, int i}.
-   */
-  static Collector<CodeBlock, ?, CodeBlock> joiningCodeBlocks(String delimiter) {
-    return Collector.of(
-        () -> new CodeBlockJoiner(delimiter, CodeBlock.builder()),
-        CodeBlockJoiner::add,
-        CodeBlockJoiner::merge,
-        CodeBlockJoiner::join);
-  }
-
   /**
    * Joins {@link CodeBlock} instances in a manner suitable for use as method parameters (or
-   * arguments). This is equivalent to {@code joiningCodeBlocks(", ")}.
+   * arguments).
    */
   static Collector<CodeBlock, ?, CodeBlock> toParametersCodeBlock() {
-    return joiningCodeBlocks(", ");
-  }
-
-  /**
-   * Joins {@link TypeName} instances into a {@link CodeBlock} that is a comma-separated list for
-   * use as type parameters or javadoc method arguments.
-   */
-  static Collector<TypeName, ?, CodeBlock> toTypeNamesCodeBlock() {
-    return Collector.of(
-        () -> new CodeBlockJoiner(", ", CodeBlock.builder()),
-        CodeBlockJoiner::addTypeName,
-        CodeBlockJoiner::merge,
-        CodeBlockJoiner::join);
+    // TODO(ronshapiro,user): consider adding zero-width spaces to help line breaking when the
```

Comment:
Not cool enough

Reply:
I guess we just have to resolve the TODO stat?